Update MemoriaCatalog.xml

yuyu-2021 opened the pull request:
Chinese Translation

Tirlititi commented:
Amazing!
I was wondering if you already finished it some time ago but didn't want to place it in the Mod Manager. I didn't know if I should contact you. Great to see that it was because you were refining it.

There are a couple of changes about the description:
- Website: it is not absolutly required although it is better to provide one. I don't know if [http://bbs.ffsky.com/forum.php?mod=viewthread&tid=2279024](http://bbs.ffsky.com/forum.php?mod=viewthread&tid=2279024) can be used for it? If I understand correctly, it is currently to view that topic without an account on FF Sky's forum, but maybe that restriction will soon be removed? Another possibility is to direct toward the historical page [http://bbs.ffsky.com/disp.aspx?ID=1651393&bid=4](http://bbs.ffsky.com/disp.aspx?ID=1651393&bid=4)
- The download URL must be a direct download link. We had an issue with Mediafire previously with another mod because the direct link expired after a long time (more than 6 months). It is relatively trustful in that regard but not totally. The following direct download link can be used at least for some time: [Direct download link](https://download1078.mediafire.com/r8jkbk1iahdgC19MXS_yYgwLe7pYwZUAjPWorCys8s5VxlLW-MnFH6tZHRQO2_zoK6i7ZzzTWTAh3jh8QXiOa2Pkz94p58YPhU-0pKZZ6fHtS--Dl1hpv1bIqc5lK_dsbm-LfMFc5uD8Lldu9Um8ovzQPcpf2gPyRGajJ9oNQ3E/36whznxe5bed6ny/Chinese+%28FFSky+Rebirth%29.rar). Sorry about that technicality.

Please tell me if these changes are OK with you or update the pull request if you want something else.

Tip: you can test what the description gives in the catalog by opening your file ``FINAL FANTASY IX/ModCatalog.xml``, add your mod description there and then flag the file as "Read only" (so the launcher will not immediately replace it with the online catalog). I did that for my test and could install the mod and play it a bit without any problem after modifying the download link.

Congrats to you and everyone involved in that project!

yuyu-2021 commented:
It's sooo good to see you like this mod and we are really happy that we can release it today haha, 

FFsky's been a very old website for years , and almost no one's keeping it for a good condition right now. Actually I find it hard to contact the administrator to create a special page that does not require an account for showing these achievements. May be I will try another way ....

Gonna give it a try haha

but it's just so awesome to release this mod hahahahaaaa , big day!

Can I just update the download link you provided to release the mod for now and then try to find another way out at the same time ?

I want thank you again for you help haha .

Check this ，Chinese (FFSky Rebirth) 's very hot right now  . All the efforts we made are worthy haha . 

https://www.bilibili.com/video/BV1wAXYYvEma/?buvid=XY9D11C9C368E92EFD6F8B6E5161B97DD3261&from_spmid=default-value&is_story_h5=false&mid=OJ%2BNijIZJchYQoFn%2BCfU6A%3D%3D&plat_id=116&share_from=ugc&share_medium=android&share_plat=android&share_session_id=cd28e3b8-78b2-485b-bc9f-567bccc1fa8a&share_source=QQ&share_tag=s_i&spmid=united.player-video-detail.0.0&timestamp=1742821536&unique_k=KNmywz3&up_id=356156951&vd_source=bf592495f2a83a462e30dcdd360eec4d

yuyu-2021 commented:
> Yes, no problem: it will always be possible to use another link in the future if needed. The topic on FF Sky is now publically available. It was restricted only for a couple of hours :) Nice video, thanks.
> 
> **Edit**: the description with these changes would be something like that:
> 
> ```
> <Mod>
> 	<Name>Chinese (FFSky Rebirth)</Name>
> 	<Version>0.9</Version>
> 	<InstallationPath>Chinese (FFSky Rebirth)</InstallationPath>
> 	<ReleaseDate>2025-03-24</ReleaseDate>
> 	<ReleaseDateOriginal>2025-03-24</ReleaseDateOriginal>
> 	<Author>Team "Tribute to FFIX cn 2008"</Author>
> 	<Description>                        水晶的记忆    十七载的传承
> 
>         ChineseFFSkyRebirth 是一款基于 Memoria 引擎开发的中文mod，全面继承了 2008 年 《最终幻想 IX》PS 版汉化版的所有元素，并在此基础上进行了进一步的优化与提升。
> 
>         本作不仅完整保留了当年汉化组的工作成果，还在Memoria引擎的帮助下进行了多项增强，使得这款经典汉化版在新技术的加持下焕发新生。
> 
>         包括：1、天幻版文本移植与改进。2、含文字的图片汉化。3、所有场景地名图片汉化且调整尺寸适应大屏幕。4、开场和结局动画的优化处理。
> 
>         我们致敬原最终幻想9汉化组的所有成员，诚挚感谢memoria引擎成员Tirlititi的无私帮助，同时也向为本mod付出热情和贡献的朋友致以最高的敬意。
> 
>         Team "Tribute to FFIX cn 2008" 祝您游戏愉快！</Description>
> 	<PatchNotes>Version 0.9 为最终幻想IX 2008年ps版汉化的移植作品，在此基础上重新制作和调整了全部图片部分，以及结局动画。</PatchNotes>
> 	<Category>Translation</Category>
> 	<Website>http://bbs.ffsky.com/forum.php?mod=viewthread&amp;tid=2279024</Website>
> 	<DownloadUrl>https://download1078.mediafire.com/r8jkbk1iahdgC19MXS_yYgwLe7pYwZUAjPWorCys8s5VxlLW-MnFH6tZHRQO2_zoK6i7ZzzTWTAh3jh8QXiOa2Pkz94p58YPhU-0pKZZ6fHtS--Dl1hpv1bIqc5lK_dsbm-LfMFc5uD8Lldu9Um8ovzQPcpf2gPyRGajJ9oNQ3E/36whznxe5bed6ny/Chinese+%28FFSky+Rebirth%29.rar</DownloadUrl>
> 	<PreviewFileUrl>http://www.duose.com/p/2007/0723/61360_5625.jpg</PreviewFileUrl>
> 	<MinimumMemoriaVersion>2024-12-04</MinimumMemoriaVersion>
> </Mod>
> ```
> 
> I can take care of these changes, or you can do it, as you wish. (I also removed the two dots in `<Author>` because I think they didn't display nicely in the launcher but it's up to you.)

sure~  thanks : ), may be We can rely this on you😁  can't even keep my open cause it's almost 12 rigth now😂😂😂

yuyu-2021 commented:
> Have a good night. Thanks again.



> Have a good night. Thanks again.

Thanks ~ The mod is in manager now !

I submitted a pull request again , replaced the download url ,changed the .RAR file with a ZIP . I believe the download function will work well now .
